src/runtime/4.8/windowsservercore-ltsc2016/Dockerfile

```diff
@@ -5,15 +5,15 @@ FROM mcr.microsoft.com/windows/servercore:ltsc2016-amd64
 ENV COMPLUS_NGenProtectedProcess_FeatureEnabled=0
 
 RUN `
-    # Install .NET 4.8
+    # Install .NET Fx 4.8
     powershell -Command `
         $ProgressPreference = 'SilentlyContinue'; `
         [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; `
         Invoke-WebRequest `
             -UseBasicParsing `
             -Uri "https://download.visualstudio.microsoft.com/download/pr/7afca223-55d2-470a-8edc-6a1739ae3252/abd170b4b0ec15ad0222a809b761a036/ndp48-x86-x64-allos-enu.exe" `
             -OutFile dotnet-framework-installer.exe `
-    && .\dotnet-framework-installer.exe /q `
+    && start /w .\dotnet-framework-installer.exe /q `
```

Comment:
This is pattern used in all other ltsc2016 Dockerfiles installing 4.x so I didn't conditionalize it here.